README.md

```diff
@@ -18,6 +18,7 @@ ReAtom is **declarative** and **reactive** state manager, designed for both simp
 - ⚡ **performance**: light tiny updates in huge state
 - 📦 **modular**: reusable instances (SSR)
 - 🗜 **small size**: [2 KB](https://bundlephobia.com/result?p=@reatom/core) gzipped
+- 🍴 **lazy atoms**: great solution for code splitting
```

Comment:
"atoms" is not a goal, it is a remedy. Maybe something like "**lazy**: posibility to add the parts of store asynchronously".

Reply:
Were such suggestions:

1. traffic optimization: solution for code splitting out of the box
2. async stores: solution for code splitting out of the box
3. lazy stores: solution for code splitting out of the box

Reply:
`**lazy**: solution for code splitting out of the box` wiil be fine